.github/workflows/notify-mm-blog.yml

```diff
@@ -19,10 +19,10 @@ jobs:
           .github/workflows/notify-mm-blog.sh
       - name: Notify channel 1
         uses: mattermost/action-mattermost-notify@master
```

Comment:
Should we pin the version to avoid breaking changes?

Reply:
Let's do that, rebasing and pushing the recommendation

Reply:
@hanzei done https://github.com/mattermost/mattermost-marketplace/compare/3bfdb9c9353eb89a9790b02cc57476da839b44f8..84010f4ad0fd719a3ecb97af082fda213d980869